client/app/email-domain/general-informations/email-domain-general-informations.html

```diff
@@ -1,125 +1,105 @@
-                                                  data-tooltip-placement="right"></span>
-                                        </strong>
-                                        <oui-spinner data-size="s" data-ng-if="ctrlTabGeneralInformations.loading.quotas"></oui-spinner>
-                                        <span data-ng-bind="(ctrlTabGeneralInformations.summary.account || '0') + ' / ' + (ctrlTabGeneralInformations.quotas.account || '0')"
-                                              data-ng-if="!ctrlTabGeneralInformations.loading.quotas"></span>
-                                    </div>
-                                </li>
-                                <li class="oui-tile__item">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block"
-                                                data-translate="email_tab_responders_quota"></strong>
-                                        <oui-spinner data-size="s" data-ng-if="ctrlTabGeneralInformations.loading.quotas"></oui-spinner>
-                                        <span data-ng-bind="(ctrlTabGeneralInformations.summary.responder || '0') + ' / ' + (ctrlTabGeneralInformations.quotas.responder || '0')"
-                                              data-ng-if="!ctrlTabGeneralInformations.loading.quotas"></span>
-                                    </div>
-                                </li>
-                                <li class="oui-tile__item">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block"
-                                                data-translate="email_tab_redirections_quota"></strong>
-                                        <oui-spinner data-size="s" data-ng-if="ctrlTabGeneralInformations.loading.quotas"></oui-spinner>
-                                        <span data-ng-bind="(ctrlTabGeneralInformations.summary.redirection || '0') + ' / ' + ((ctrlTabGeneralInformations.quotas.redirection + ctrlTabGeneralInformations.quotas.alias) || '0')"
-                                              data-ng-if="!ctrlTabGeneralInformations.loading.quotas"></span>
-                                    </div>
-                                </li>
-                            </ul>
+                        <div>
+                            <span data-ng-bind="ctrlTabGeneralInformations.serviceInfos.contactTech"></span>
+                            <small data-translate="global_technical"></small>
                         </div>
-                    </div>
-                </div>
-            </div>
+                        <div>
+                            <span data-ng-bind="ctrlTabGeneralInformations.serviceInfos.contactBilling"></span>
+                            <small data-translate="global_billing"></small>
```

Comment:
Only use `small` for fine prints. Use CSS if you want the text to be smaller

Reply:
The small text styling has been moved to css